pkg/lint/testdata/files/no-epoch.yaml

```diff
@@ -0,0 +1,9 @@
+package:
+  name: no-epoch
+  version: 1.2.3
+  description: "a package that fails as no epoch"
+  copyright:
+    - paths:
```

Comment:
You can remove these, which might trigger other lint checks and cause the test to pass erroneously

Reply:
Just to confirm which can be removed?  When I remove anything other than the `attestation: TODO` the other lint check fails:
```
valid-copyright-header every package should have a valid copyright header INFO
```